app/serializers/provider_serializer.rb

```diff
@@ -33,7 +33,14 @@ class ProviderSerializer < ActiveModel::Serializer
   attributes :institution_code, :institution_name, :institution_type, :accrediting_provider,
              :address1, :address2, :address3, :address4, :postcode, :region_code, :scheme_member,
              :contact_name, :email, :telephone, :recruitment_cycle, :utt_application_alerts,
-             :type_of_gt12
+             :type_of_gt12, :application_alert_recipient
+
+  attribute :contacts do
+    object.contacts.map { |c| c.attributes.slice('type', 'name', 'email', 'telephone') } + [{
+      type: 'application_alert_recipient',
+      email: object.ucas_preferences.application_alert_email,
+      }]
```

Comment:
I know I'm the one that contributed this bit of mess, but how about we use a transient variable here to help explain what `object.contacts.map { ... }` is all about.

It would probably also help to have a comment to explain what's going on here.

Reply:
Okay have done! Thanks for all the feedback 👍